static/translations/messages_en.json

```diff
@@ -141,7 +141,7 @@
   "Learn more": "Learn more",
   "Full profile": "Full profile",
   "Settings": "Settings",
-  "Night mode": "Night mode",
+  "Dark mode": "Dark mode",
```

Comment:
It looks like this string doesn't end up appearing in the UI; so we can just delete it.